diff --git a/tools_webrtc/libs/generate_licenses.py b/tools_webrtc/libs/generate_licenses.py
index f33c050291..6fa6520a8e 100755
--- a/tools_webrtc/libs/generate_licenses.py
+++ b/tools_webrtc/libs/generate_licenses.py
@@ -29,6 +29,7 @@ import logging
 import os
 import re
 import subprocess
+import codecs
 
 # Third_party library to licences mapping. Keys are names of the libraries
 # (right after the `third_party/` prefix)
@@ -217,8 +218,8 @@ class LicenseBuilder(object):
         logging.info('List of licenses: %s', ', '.join(license_libs))
 
         # Generate markdown.
-        output_license_file = open(os.path.join(output_dir, 'LICENSE.md'),
-                                   'w+')
+        output_license_file = codecs.open(os.path.join(output_dir, 'LICENSE.md'),
+                                   'w+', encoding='utf-8')
         for license_lib in license_libs:
             if len(self.common_licenses_dict[license_lib]) == 0:
                 logging.info(
@@ -230,7 +231,7 @@ class LicenseBuilder(object):
             output_license_file.write('```\n')
             for path in self.common_licenses_dict[license_lib]:
                 license_path = os.path.join(WEBRTC_ROOT, path)
-                with open(license_path, 'r') as license_file:
+                with codecs.open(license_path, 'r', encoding='utf-8') as license_file:
                     license_text = cgi.escape(license_file.read(), quote=True)
                     output_license_file.write(license_text)
                     output_license_file.write('\n')
